### PR TITLE
Add github link doc

### LINKED ISSUE #87: Missing github repo links doc

### PR DESCRIPTION
**Pull request by johnsudaar**
Fix #87

### CODE REVIEW

File: markdown/github-repo-links.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+--- row ---
+
+`POST https://api.scalingo.com/v1/apps/:app_id/github_repo_links`
+
+### Parameters
+
+* `github_repo_link.github_source` - required: Github repository to link
+* `github_repo_link.github_branch` - optional: Branch used for the auto deploy feature
+* `github_repo_link.auto_deploy_enabled` - optional: Trigger a new deployment when changes are pushed to the selected branch
```
Comment (EtienneM) [inline markdown/github-repo-links.md]: Replace "the selected branch" with "branch specified with the `github_repo_link.github_branch` parameter"

File: markdown/github-repo-links.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+* `github_repo_link.deploy_review_apps_enabled` - optional: Enable the [review app](https://doc.scalingo.com/platform/app/review-apps#top-of-page) feature
+* `github_repo_link.delete_on_close_enabled` - optional: Delete review app when the linked PR is closed
+* `github_repo_link.hours_before_delete_on_close` - optional: Time to wait before deleting a review with a closed PR
```
Comment (EtienneM) [inline markdown/github-repo-links.md]: "review with a closed PR" -> "review app associated to a closed PR"

File: markdown/github-repo-links.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+`POST https://api.scalingo.com/v1/apps/:app_id/github_repo_links`
+
+### Parameters
+
+* `github_repo_link.github_source` - required: Github repository to link
+* `github_repo_link.github_branch` - optional: Branch used for the auto deploy feature
+* `github_repo_link.auto_deploy_enabled` - optional: Trigger a new deployment when changes are pushed to the selected branch
+* `github_repo_link.deploy_review_apps_enabled` - optional: Enable the [review app](https://doc.scalingo.com/platform/app/review-apps#top-of-page) feature
+* `github_repo_link.delete_on_close_enabled` - optional: Delete review app when the linked PR is closed
```
Comment (EtienneM) [inline markdown/github-repo-links.md]: Is "PR" initials that people know? Shouldn't we use "pull request"?

File: markdown/github-repo-links.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
+||| col |||
+
+Example object:
+
+```json
+{
+  "id":"1d6e59a0-5377-11e8-90e6-0242ac110052",
+  "linker": {
+    "id":"us-aa263090-9a23-81f5-8c7c-5fd9a50a8fa8"
```
Comment (Soulou) [inline markdown/github-repo-links.md]: invalid JSON here :)

File: markdown/github-repo-links.md (hunk 3 above)
Comment (Soulou) [reply markdown/github-repo-links.md]: I think we should use pull request

**Pull request closed without merging**